install/do_owner.php

```diff
@@ -13,8 +15,13 @@
 
     $mysqli = new mysqli($db['account']['hostname'], $db['account']['username'], $db['account']['password'], $db['account']['database'], $db['account']['port']);
 
-    $query = mysqli_query($mysqli, "SELECT username from accounts WHERE username = ".$accname."");
-    if (mysqli_num_rows($query) === 0) {
+    if (mysqli_connect_errno()) {
+        echo json_encode(array("success" => false, "message" => $mysqli->connect_error));
+        exit();
+    }
+
+    $query = mysqli_query($mysqli, "SELECT username from account WHERE username = '".$accname."' LIMIT 1");
```

Comment:
You could also not do the `LIMIT` and then throw an error if more than 1 result is returned because that shouldn't be possible